TestApp/ios/Podfile.lock

```diff
@@ -401,6 +407,7 @@ SPEC CHECKSUMS:
   RNCAsyncStorage: 5ae4d57458804e99f73d427214442a6b10a53856
   RNFS: 795866388a01de4e470f3b1041a99cf20a343844
   RNGestureHandler: a4ddde1ffc6e590c8127b8b7eabfdade45475c74
+  RNScreens: 9d8a2900cd4f967af70dbfed74b64f77fbb6600a
```

Comment:
RNScreens was added automatically. Does it need to be deleted?

Reply:
@AnastasiaKubova please review it, Thanks.

Reply:
I think it's ok